added downScaling script for utilitypv

Kaviyarsan2004 opened the pull request.

Kaviyarsan2004 committed:
added downScaling script for utilitypv

gschivley commented:
Hi, thanks for taking a pass at the downscaling script. I looked over `downScaling_script.py` and it appears to pick the lowest LCOE CPAs in a cluster of sites. Based on my understanding of @nspatank's R code, the downscaling needs to account for how CPAs have been population de-rated, apply some technology-specific factor that determines how many CPA's are in an average cluster, and randomly select clusters of CPAs. This is what I did in the [`random_select_cluster_cpas`](https://github.com/gschivley/downscaling-tool/blob/4392747c18cbd848456ce5b16451cd74e7e82240/greg_src/downscale_results.py#L186) function.

Can you confer with Neha and add this type of functionality? The eventual plan is also to make it a tool where users can run something from command line and generate all downscaled results -- including the option to make maps or export a shapefile if users want. I can bundle it all together, just letting you know where this is going.

Kaviyarsan2004 committed:
updated down_scaling script

Kaviyarsan2004 committed:
Remove embedded repository from index

Kaviyarsan2004 committed:
added output sample

Kaviyarsan2004 committed:
added plot

Kaviyarsan2004 committed:
removed script

Kaviyarsan2004 committed:
rm folder

Kaviyarsan2004 commented:
Hi, Thanks for reviewing the downscaling script. I've now added the option to select CPAs by population de-rating, as well as the capability for users to generate a shapefile. At the end of the process, users will receive two files: one containing all the CPAs that satisfy the required capacity for each cluster, and another with the population de-rated CPAs, implemented similarly to how you did it in the random_select_cluster_cpas function in the 'downscaling.py' file, Let me know if there are any further adjustments needed!

Kaviyarsan2004 committed:
Update downScaling_script.py
